docs/src/modules/javascript/pages/actions-publishing-subscribing.adoc

```diff
@@ -0,0 +1,94 @@
+include::ROOT:partial$include.adoc[]
+
+= Publishing and Subscribing with Actions
+
+A very common use case when building Microservices is to publish and subscribe to a stream of events. The source of events can be the journal of an event sourced entity, the value entity state changes, a https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] or Apache Kafka topic for asynchronous messaging between services.
+
+With Actions you can:
+
+- subscribe to events emitted by an event sourced entity within the same service.
+- subscribe to state changes emitted by a value entity within the same service.
+- subscribe to external events from https://cloud.google.com/pubsub/docs/overview[Google Cloud Pub/Sub] or Apache Kafka.
+- publish events to a Google Cloud Pub/Sub or Apache Kafka topic.
+
+Messages are guaranteed to be delivered at least once. This means that receivers must be able to handle duplicate messages.
+
+== Publishing Entity events to a Topic
+
+The Event Sourced Entity journal contains events that capture all state changes. By subscribing to the journal, with the Event Sourced Entity type name, another component can receive all events emitted of that type.
+
+.Use case: send asynchronous messages
+****
+By combining the processing of a journal with publishing to topics (see below), a service can trigger other services asynchronously via messaging.
+To achieve this, create an xref:actions.adoc[Action] that subscribes to a journal and publishes messages to a topic.
+****
+
+.Use case: external calls
+****
+A service might need to trigger other systems when certain events happened to an Entity. An Action can be connected to the Entity's journal and react on certain events to issue calls (eg. via HTTP or gRPC).
+To achieve this, create an xref:actions.adoc[Action] that subscribes to a journal and let the implementation call other services.
+****
+
+== Subscribing to state changes from a Value Entity
+
+To subscribe to an Event Sourced Entity log, define Protobuf rpc methods for each Journal event that you want to receive. Annotate these methods with the `(kalix.method).eventing` annotation and specify the Entity type name of the Event Sourced Entity.
```

Comment:
Title (`state changes from a Value Entity`) and content (`Event Sourced Entity log`) don't align here.